Proposal for the basic structure of a simulation study.md

```diff
@@ -0,0 +1,49 @@
+# Proposal for the basic structure of a simulation study
+
+### Final Structure
+
+1. *Defining aims & objectives (including hypotheses)*
+2. *Simulation procedure /Design & Data-generation mechanism*
+3. *Methods selection (including software and estimands)*
+4. *Optional: Model specification*
+5. *Selection of performance measures*
+6. *Coding & execution*
+7. *Analyzing results*
+8. *Presentation/ Reporting*
+
+Basis: literature research which included Papers from psychology, medicine, statistics and engineering.
+
+Goals: Propose a structure that… 
+
+- finds a balance of detail and abstraction along the entire process of a simulation study in our field of interest, from planning to presentation
+- is compatible with other proposed structures.
+
+The Papers displayed heterogeneity in their detail, overall and in specific areas of the simulation process, firstly, with regards to the use cases the authors are interested in. As we are interested in conducting a simulation study in the context of psychological methods, suggested structures by authors with engineering background were deemed to be less relevant (Law & Comas, 1991). Other papers, with a similar background in authorship, were explicitly meant to be interpreted broadly, which lead to a broad coverage of the topic as well, lacking relevant detail for our application (Shannon, 1992; Balco, 1990).
+
+Besides these suggestions, other papers proposed sensible structures but for different purposes, for example to provide reporting guidelines or to advocate for the preemptive formulation of a simulation plan (Boomsma, 2013; Monks et al., 2019; Smith & Marshall, 2010). Two papers proposed an applicable structure but had a strong focus on the execution and coding of a simulation, on the cost of less detail on planning and preparation of the study (Paxton et al., 2001; Harwell et al., 1996). Similarly, a paper from Burton et al. (2006) proposed a good structure as well, but put a lot of focus on method-related decisions in the simulation procedure up until the evaluation of the results (Step 2-7). 
```

Comment:
I somewhat agree that a more explicit separation between the coding phase and planing phase is sensible. For this I might suggest to treat the "Data generation mechanism step" in this structure (or the designing specific experimental conditions) as suggested by Paxton et al. (2006) as to be written in a semi-standardized pseudo code syntax to 1) keep the planing phase separate from the actual code implementation for further isolation of "raw" decision processes from "arbitrary" coding option while 2) enhancing post hoc comparability and workflow optimazation when presenting our approach as a (novel) methodological approach.

Reply:
I agree and will add this explicitly in the step!